src/perl/vdotest/VDOTest/Dmsetup.pm

```diff
@@ -64,6 +65,35 @@ sub testBasicOps {
   assertNe("", $EVAL_ERROR, "unknown message failed to generate an error");
 }
 
+###############################################################################
+# Test dmsetup message for displaying config information
+#
+##
+sub testConfig {
+  my ($self) = assertNumArgs(1, @_);
+  my $device = $self->getDevice();
+  my $path = $device->getVDODevicePath();
+
+  my $output = $device->runOnHost("sudo dmsetup message $path 0 config");
+  $output = "config : " . $output;
+  my $yaml = yamlStringToHash($output);
+
+  my $slabBits = $self->{slabBits};
+  if (!defined($slabBits)) {
+    # default slab bits from vdoformat
+    $slabBits = 19;
+  }
```

Comment:
Can you make sure this is testing with a non-default number of slab bits? It's worth making sure we're actually reading the config.

Reply:
Ok. I can make another test.